python/ray/experimental/gpu_object_manager/gpu_object_manager.py

```diff
@@ -218,6 +252,19 @@ def trigger_out_of_band_tensor_transfer(
 
             src_actor = gpu_object_meta.src_actor
             tensor_meta = gpu_object_meta.tensor_meta
+            if gpu_object_meta.tensor_transport_backend == Backend.NIXL:
+                if src_actor == dst_actor:
+                    # Intra-actor communication
+                    continue
+                self._send_gpu_object("nixl", src_actor, arg.hex(), 0)
```

Comment:
Why do we need to call a task on the sender actor?

Reply:
Updated